README.rst

```diff
@@ -676,3 +676,33 @@ page rendering and painting.
 .. _`varies by OS` : https://en.wikipedia.org/wiki/Hosts_%28file%29#Location_in_the_file_system
 .. _`简体中文`: https://github.com/skyline75489/what-happens-when-zh_CN
 .. _`downgrade attack`: http://en.wikipedia.org/wiki/SSL_stripping
+
+Turn On a linux Box
+-------------------
+
+There are 6 elements involved on the Boot process of a Linux box:
+
+1. **Bios**
+  * Perform system integrity checks.
+  * Search, Load and execute the Boot loader program.
+  * Scans for devices (Hard Drives, cd-rom, Floppy disk) once it detects the boot loader it loads it to memory.
+
+2. **MBR**
```

Comment:
Actually yes, but wanted to do the more generic one. Would it make sense to add both or just GPT as the latest?